src/main/java/com/ticketfinder/Seat.java

```diff
@@ -0,0 +1,28 @@
+package com.ticketfinder;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Seat {
+
+    private UUID id;
+    private String description;
+    private int price;
+    private boolean reserved;
+    private String nameAndSurname;
```

Comment:
If you want to store a name and surname please do that in two fields, or even create a separate class with thees two fields. 

There is a good practice to don't use a "and" keyword, becouse it suggest that you want to do two things at the same times. The better approach is to split it into two separate things